ibm/service/metricsrouter/resource_ibm_metrics_router_route.go

```diff
@@ -0,0 +1,381 @@
+// Copyright IBM Corp. 2023 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package metricsrouter
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/metricsrouterv3"
+)
+
+func ResourceIBMMetricsRouterRoute() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMMetricsRouterRouteCreate,
+		ReadContext:   resourceIBMMetricsRouterRouteRead,
+		UpdateContext: resourceIBMMetricsRouterRouteUpdate,
+		DeleteContext: resourceIBMMetricsRouterRouteDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validate.InvokeValidator("ibm_metrics_router_route", "name"),
+				Description:  "The name of the route. The name must be 1000 characters or less and cannot include any special characters other than `(space) - . _ :`. Do not include any personal identifying information (PII) in any resource names.",
+			},
+			"rules": &schema.Schema{
+				Type:        schema.TypeList,
+				Required:    true,
+				Description: "Routing rules that will be evaluated in their order of the array.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"action": &schema.Schema{
+							Type:         schema.TypeString,
+							Optional:     true,
+							Description:  "The action if the inclusion_filters matches, default is `send` action.",
```

Comment:
If it has default value then it shd be marked as both optinal and computed true?

Reply:
done